src/app/projects/project-progress-dashboard/project-progress-dashboard.tpl.html

```diff
@@ -39,55 +44,70 @@ <h3 class="panel-title">Target Grade</h3>
           </div>
-          <div class="panel-body">
-            <div class="burndown-chart" ng-if="currentVisualisation == 'burndown'">
-              <progress-burndown-chart project="project" unit="unit"></progress-burndown-chart>
-            </div>
-            <div ng-if="currentVisualisation == 'taskSummary'">
-              <student-task-status-pie-chart project="project"></student-task-status-pie-chart>
-            </div>
+          <div ng-show="currentVisualisation == 'taskSummary'">
+            <h3 class="panel-title">Task Summary Chart</h3>
+            Summary of each of your task statuses
           </div>
-          <div class="panel-footer text-center">
-            <div ng-show="currentVisualisation == 'taskSummary'">
-              You have completed <strong>{{taskStats.numberOfTasksCompleted}}</strong> tasks and have <strong>{{taskStats.numberOfTasksRemaining}}</strong> left to complete to achieve a <strong>{{grades[project.target_grade]}}</strong>.
-            </div>
-            <div ng-show="currentVisualisation == 'burndown'">
-              Aim to keep your <strong style="color: #e01b5d">Complete</strong> line close to or ahead of the <strong class="text-muted">Target</strong> line to keep on track.
-            </div>
+        </div>
+        <!--/headings-->
+        <div class="pull-right text-right">
+          <div class="btn-group">
+            <label class="btn btn-default text-center" btn-radio="'burndown'" ng-model="currentVisualisation">
+              <i class="fa fa-line-chart"></i>
+            </label>
+            <label class="btn btn-default text-center" btn-radio="'taskSummary'" ng-model="currentVisualisation">
+              <i class="fa fa-pie-chart"></i>
+            </label>
           </div>
         </div>
+        <!--/button-switch-->
+      </div>
+      <div class="panel-body">
+        <div class="burndown-chart" ng-if="currentVisualisation == 'burndown'">
+          <progress-burndown-chart project="project" unit="unit"></progress-burndown-chart>
+        </div>
+        <div ng-if="currentVisualisation == 'taskSummary'">
+          <student-task-status-pie-chart project="project"></student-task-status-pie-chart>
+        </div>
+      </div>
+      <div class="panel-footer text-center">
+        <div ng-show="currentVisualisation == 'taskSummary'">
+          You have completed
+          <strong>{{taskStats.numberOfTasksCompleted}}</strong>
+          tasks and have
+          <strong>{{taskStats.numberOfTasksRemaining}}</strong>
+          left to complete to achieve a
+          <strong>{{grades[project.target_grade]}}</strong>.
+        </div>
+        <div ng-show="currentVisualisation == 'burndown'">
+          Aim to keep your
+          <strong style="color: #e01b5d">Complete</strong>
+          line close to or ahead of the
+          <strong class="text-muted">Target</strong>
+          line to keep on track.
+        </div>
       </div>
-    </div><!--primary-panel-->
```

Comment:
Did you use a linter here? We would like to keep `<!-->` comments on the same div line as it's consistently like that throughout the project :)